torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1277,6 +1277,12 @@ def statically_known_true(x: Union[bool, SymBool]) -> bool:
     return x
 
 
+# a or b is eager, meaning that (a==1 or True) will specialize a.
+# but operator.or_(a==1, True) generates sympy expression and avoid
+# the specialization.
```

Comment:
It took me some time to understand this. The key is that operator.or_ will respect the dunder method `__or__` on symnode instead of eagerly evaluating. Perhaps you can update the comment to reflect this?

Reply:
Agreed - Definitely need to explain the difference between `or` and `operator.or_` for symnodes...

Reply:
i will add.